### PR TITLE
Threading and logging improvements

### PR DESCRIPTION
**Pull request by djesic**

### CODE REVIEW

File: test/test_wolk_connect.py [hunk 1]
```diff
@@ -705,6 +705,30 @@ def test_add_sensor_readings(self):
         wolk_device.add_sensor_readings({"R": 1})
         wolk_device.message_queue.put.assert_called_once()
 
+    def test_add_sensor_readings_historical(self):
```
Comment (srdjastankovic) [inline test/test_wolk_connect.py:708]: Why you adding into the Python Connector historical readings?

Comment (djesic) [reply test/test_wolk_connect.py:708]: To support the protocol option to publish multiple data of a single sensor in a single payload.

Comment (srdjastankovic) [reply test/test_wolk_connect.py:708]: than it isn't historical, it is multuple sensor readings. Do we have this in the other wolkconnectors?

Comment (djesic) [reply test/test_wolk_connect.py:708]: That is just how i named the test.
The API is `add_sensor_readings` which previously only supported the format for JSON_MULTI style sensor reading messages.

**Review by srdjastankovic — APPROVED** [review]

**Pull request merged**